core/src/main/java/pl/agh/edu/actor/TextActors/ValueTag.java

```diff
@@ -0,0 +1,94 @@
+package pl.agh.edu.actor.TextActors;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.NinePatch;
+import com.badlogic.gdx.scenes.scene2d.ui.*;
+import com.badlogic.gdx.utils.Align;
+
+import pl.agh.edu.actor.HotelSkin;
+import pl.agh.edu.config.GraphicConfig;
+import pl.agh.edu.language.LanguageChangeListener;
+import pl.agh.edu.language.LanguageManager;
+
+public class ValueTag extends Table implements LanguageChangeListener {
+	private final Skin skin;
+	private final String languagePath;
+	private Label tagLabel;
+	private Label valueLabel;
+
+	public ValueTag(String languagePath, String value) {
+		this.languagePath = languagePath;
+		this.skin = HotelSkin.getInstance();
+		Stack componentStack = new Stack();
+
+		add(componentStack).height(ValueTagStyle.getHeight()).width(ValueTagStyle.getWidth());
+		NinePatch background = skin.getPatch("valuetag-background");
```

Comment:
zamiast owrapowania wszystkiego w stacka i dodatkową tabele można użyć
```java
this.setBackground(new NinePatchDrawable(HotelSkin.getInstance().getPatch("rating-background")));
```

Reply:
Jest tylko taki problem że potem nie mogę ustawić bezpośrednio rozmiarów tablicy, jakbym miał jedno table - musiałbym w test frame zrobić to za pomocą  add().width().height(). To co ci napisałem w pr z ratingiem. Bo rozmiar tablicy zależy od rozmiaru contentu.